stepfile/testdata/lawhelp.star

```diff
@@ -0,0 +1,135 @@
+load("http.star", "http")
+load("bsoup.star", "bsoup")
+
+baseUrl = "http://www.lawhelp.org"
```

Comment:
This is a great practical example, and nicely shows a division between downloading and transforming (tho the transform is very minimal). However, I'm a bit wary of hitting a real server in order to run a test. For one thing, it becomes impossible to get tests to pass when offline, or if the server is otherwise unavailable. It makes me wish that the starlark environment had a built-in mock service for http that could use hard-coded string values just for tests.

Reply:
your broader point about keeping starlark out of this repo is a good one. This code isn't actually executed, and the test is just working the step parsing from the file. We could do that without stalark, or code that looks like starlark, to avoid confusion around what the test is doing